### PR TITLE
Add instructions for building the samples on non license managed platforms

### PR DESCRIPTION
**Pull request by datalogics-robl**
These instructions explain how to run the samples on a non-license managed platform. The README is updated to link to this file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +31,12 @@ Licensing and Pricing options are customized to your usage and requirements. For
 - Search for and replace or edit text
 
 ## Running the Samples
-- Clone this repository to your system (64-bit Intel Windows, 64-bit Intel Linux, or Apple Silicon)
+**For 64-bit Intel Windows, 64-bit Intel Linux, or Apple Silicon systems:**
```
Comment (datalogics-josepha) [inline README.md:34]: let's call it macARM to be consistent with the rest of the product naming

File: using_on_other_platforms.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+# Using APDFL and Maven on other platforms
+
+## Installing APDFL for other platforms
+
+[Maven Central](https://central.sonatype.com/artifact/com.datalogics.pdfl/pdfl) offers self-activating license managed APDFL packages for 64-bit Windows, 64-bit Intel Linux, and Mac systems
+running Apple Silicon. The samples in this repo will automatically download the APDFL package for your platform via Maven and obtain a 30 day trial license.
```
Comment (datalogics-josepha) [inline using_on_other_platforms.md:6]: why is this on a new line?

File: using_on_other_platforms.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+Customers can also obtain APDFL without license management for the same three platforms, plus those listed below. You will recieve a ZIP file with the APDFL components for your platform,
+plus a POM file (`create-artifacts.xml`) that you can use to install APDFL to your local Maven cache, or deploy to private Maven repo on your network. To install the components to your
```
Comment (datalogics-josepha) [inline using_on_other_platforms.md:9]: why is this on a new line?

**Comment by datalogics-josepha** [comment]
the name of the readme refers to other platforms, versus what?

File: using_on_other_platforms.md (hunk 2 above)
Comment (datalogics-josepha) [inline using_on_other_platforms.md:6]: let's change this to macARM as mentioned above

Comment (datalogics-robl) [reply using_on_other_platforms.md:6]: I broke the lines in my editor where it made sense. I agree that's arbitrary, but my understanding is that Markdown renderers ignore line breaks in text, so putting them in prevents having to scroll sideways during editing, and doesn't affect what customers see.

**Pull request merged**